### DISCUSSION TITLE
Namespace not specified error removed

### DISCUSSION THREAD

**Discussion by mbaykara** [fluxcd/flux2#4525] [Q&A] [answered]
Hi @Team 

In the latest release, if the namespace is omitted from HelmRelease, the error is unclear and lacks verbosity in solving it. Previously, the error was self-explanatory, stating 'namespace is not specified,' allowing users to easily identify the issue. However, in the current release, the error message is as follows:

`HelmRelease/webserver not found: the server could not find the requested resource (patch helmreleases.helm.toolkit.fluxcd.io webserver)`

does not point the issue well. What is the motivation behind this change? Thank you!

**Comment by stefanprodan** [comment, marked as answer]
Thanks for reporting this, I have open an issue to track this bug https://github.com/fluxcd/pkg/issues/716
